plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -2,100 +2,55 @@
 package azure_data_explorer
 
 import (
-	"bytes"
-	"context"
 	_ "embed"
-	"errors"
 	"fmt"
-	"strings"
 	"time"
 
-	"github.com/Azure/azure-kusto-go/kusto"
-	kustoerrors "github.com/Azure/azure-kusto-go/kusto/data/errors"
 	"github.com/Azure/azure-kusto-go/kusto/ingest"
-	"github.com/Azure/azure-kusto-go/kusto/kql"
+	"github.com/influxdata/telegraf/config"
+	adx_common "github.com/influxdata/telegraf/plugins/common/adx"
```

Comment:
We do have the unwritten convention of 
```suggestion
	common_adx "github.com/influxdata/telegraf/plugins/common/adx"
```
but it would also be okay to not use an alias as I think we don't have a collision...